truffle-config.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
       host: "127.0.0.1",
       port: 8545,
       network_id: "*",
-      gas: 800000000,
+      gas: 80000000000000000,
```

Comment:
@xiangformless Is this required?

Reply:
without this change, I got local test failure like below. 
![image](https://github.com/user-attachments/assets/2b879d23-7243-42b3-b37b-59edfd82af0a)

Reply:
@xiangformless SG let's keep.